assemble.bat

```diff
@@ -1,3 +1,5 @@
+del bin\*.o /q
```

Comment:
The .o files are already deleted at the end of the script.
It's the .a files you need to delete (riscv32i-unknown-none-elf.a / riscv32if-unknown-none-elf.a)

Reply:
🤦 oh yes - sorry